src/components/data/DataTable/TableDataCell.tsx

```diff
@@ -22,11 +24,24 @@ export interface TableDataCellProps {
      * style prop passed to td element
      */
     style?: CSSProperties;
+
+    /**
+     * boolean toggle to vertically align the data in cell
+     */
+    verticallyAligned?: boolean;
 }
 
-const TableDataCell: FunctionComponent<TableDataCellProps> = ({ children, className, rowSpan, colSpan, style }) => {
+const TableDataCell: FunctionComponent<TableDataCellProps> = ({
+    children,
+    className = '',
+    rowSpan,
+    colSpan,
+    style,
+    verticallyAligned
```

Comment:
`verticallyAligned` as undefined will be falsy, but maybe for better clarity and explicity we should add default value for it and have:
```suggestion
    verticallyAligned = false
```
?